changelog/fragments/1699604913-remove-assetbeat-dependency.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: deprecation
+
+# Change summary; a 80ish characters long description of the change.
+summary: remove assetbeat dependency
```

Comment:
You don't need a changelog fragment for this since it isn't a user facing change, you can add the skip-changelog label to bypass that check.